blueprints/component-test/index.js

```diff
@@ -22,13 +39,41 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testTypeDefinition = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  // Provide a template (string or precompiled) for this.render()" + EOL +
+      "  this.render();" + EOL + EOL +
```

Comment:
Unfortunately not since `ember-test-helpers` tries to compile a passed string via [`Ember.Handlebars.precompile()`](http://git.io/vIXtj) which will :boom: ...